arch/cortex-m/src/systick.rs

```diff
@@ -132,6 +132,13 @@ impl kernel::SysTick for SysTick {
         value > tics
     }
 
+    fn get_value(&self) -> u32 {
+        let tics = SYSTICK_BASE.syst_cvr.read(CurrentValue::CURRENT);
+        //let hertz = self.hertz();
+        //tics * 1_000_000 / hertz
```

Comment:
```suggestion
```

?

Reply:
woah, those were supposed to be included. I took them out when benchmarking this function and guess I forgot to bring them back. good catch.